portfolio/.theia/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think you can probably add this file to your `.gitignore` file? I think it's a settings file for the editor that you wouldn't need in your repo. It's not hurting anything though, so it's up to you.